tests/unit/orchestrator/test_artifacts.py

```
@@ -210,3 +212,5 @@ def test_confidence_metric_to_json_result(self):
         assert json_result.min == 95.0
         assert json_result.max == 105.0
         assert json_result.unit == "ms"
+
+    async def test_write_detailed_json(self, tmp_path):        assert data["aiperf_version"] == "unknown"
```

Ruff (0.15.5)

[error] Undefined name `data`

(F821)